src/components/WhyTechSVG/ComputerSVG.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import computerSVG from '../../images/about-us/why-tech/computer.svg';
+
+export default function ComputerSVG(){
+    return <img style={{width: '300px', height: '300px'}} alt="city" src={computerSVG} />
```

Comment:
can we directly use <svg> here rather than <img> so that we can ;et svg handle the responsivenss (otherwise img width/height would need to be managed)